bricksrc/ontology.py

```diff
@@ -8,13 +9,25 @@
 ontology = {
     DCTERMS.creator: [
         {
+            RDF.type: SDO.Person,
             SDO.email: Literal("gtfierro@cs.berkeley.edu"),
             SDO.name: Literal("Gabe Fierro"),
         },
-        {SDO.email: Literal("jbkoh@eng.ucsd.edu"), SDO.name: Literal("Jason Koh")},
+        {
+            RDF.type: SDO.Person,
+            SDO.email: Literal("jbkoh@eng.ucsd.edu"),
+            SDO.name: Literal("Jason Koh"),
+        },
     ],
     DCTERMS.license: URIRef("https://github.com/BrickSchema/brick/blob/master/LICENSE"),
-    DCTERMS.version: Literal(BRICK_FULL_VERSION),
+    DCTERMS.issued: Literal("2016-11"),
```

Comment:
dcterms:issued should probably be in the form "YYYY-MM-DD".

Reply:
The abbreviated dates are technically allowed [according to the purl docs](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http%3a%2f%2fpurl.org%2fdc%2felements%2f1.1%2fdate) but I can update it to a full date